lua/lazyvim/plugins/extras/lang/prisma.lua

```diff
@@ -0,0 +1,19 @@
+return {
+  recommended = {
+    ft = "prisma",
+  },
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = {
+      ensure_installed = { "prisma" },
+    },
+  },
+  {
+    "neovim/nvim-lspconfig",
+    opts = {
+      servers = {
+        prismals = {},
```

Comment:
I don't really know anything about prisma, but are there any additional configurations to the lsp that you think would be nice? If the default values are good, then thats perfect!
Here is the link https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#prismals

Reply:
Defaults work fine in this case